wiki/Documentation-Single-Source-of-Truth.md

```diff
@@ -77,7 +77,7 @@ No intermediate events are currently supported.
 
 ### Overview SSOT hierarchy
 
-![](https://i.imgur.com/7yOqDZv.png)
+![image](https://user-images.githubusercontent.com/36270527/120194098-705e4b00-c21d-11eb-9301-937447369e93.png)
```

Comment:
Can you have a look at the heading of `**parameter object extending the INSTRUCTION:**`? It's a few lines below. I'd like to change it to:

`**Parameter Object Extending the INSTRUCTION Element**`